Resources/Prototypes/Loadouts/Jobs/Epistemics/chaplain.yml

```
@@ -47,6 +47,19 @@
   items:
     - BoxCandle
 
+- type: loadout
+  id: LoadoutChaplainEquipmentOuijiBoard
+  category: JobsEpistemicsChaplain
+  cost: 5
```

```suggestion
  cost: 13
```
the most expensive chaplain loadout is a whopping 4/14 points
you should have to break the bank to get this thing